examples/ron/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "ron"
+version = "0.0.0"
+workspace = "../../"
+edition = "2018"
+publish = false
+
+[dependencies]
+rocket = { path = "../../core/lib" }
+serde = "1.0"
+ron = "0.5.1"
```

Comment:
We can remove this line, since it's now the wrong version of `ron` and the example doesn't actually need it.